.autover/changes/A37D7270-9F43-4933-9FAC-BFBBBC4670CB.json

```diff
@@ -0,0 +1,12 @@
+{
+	"Projects": [
+    {
+      "Name": "Amazon.Extensions.S3.Encryption",
+      "Type": "Patch",
+      "ChangelogMessages": [
+        "Mark the assembly trimmable",
```

Comment:
Is it worth calling out the replacement of LitJson? Possible performance improvements?

Reply:
I can't see there being any performance benefit and it is underlying implementation detail that I don't think users would care.